RecoBTag/ONNXRuntime/plugins/ParticleNetSonicJetTagsProducer.cc

```diff
@@ -131,26 +140,74 @@ void ParticleNetSonicJetTagsProducer::acquire(edm::Event const &iEvent, edm::Eve
   iEvent.getByToken(src_, tag_infos);
   client_->setBatchSize(tag_infos->size());
   skippedInference_ = false;
+
+  emptyJets_.clear();
+
+  if (!countedInputs) {
```

Comment:
Hi Kevin, do you mean moving this to the ParticleNetConstructor function here https://github.com/cms-sw/cmssw/blob/f9321048eedd4c8b4bfc5fbe9b8cd81eac250417/RecoBTag/FeatureTools/src/deep_helpers.cc#L260?
I need the emptyJets_ vector to be cleared for every event, which is why I added it here, instead of just the call at line 97, which you said was not needed

Reply:
This comment was about initializing the constants that happens in this if block. (Indeed the clear() call should happen in acquire().)

Reply:
Ahhh I see what you mean.  Yes, moved that block into the constructor